internal/publish/database/exposure.go

```diff
@@ -367,17 +367,38 @@ func executeReviseExposure(ctx context.Context, tx pgx.Tx, stmtName string, exp
 	return nil
 }
 
-// InsertAndReviseExposures transactionally revises and inserts a set of keys as necessary.
-func (db *PublishDB) InsertAndReviseExposures(ctx context.Context, incoming []*model.Exposure, token *pb.RevisionTokenData, tokenRequired bool) (int, error) {
-	logger := logging.FromContext(ctx)
+// InsertAndReviseExposuresRequest is used as input to InsertAndReviseExposures.
+type InsertAndReviseExposuresRequest struct {
```

Comment:
The method signature was getting too long and difficult to understand what params meant what.